src/Cameras/babylon.freeCamera.ts

```diff
@@ -102,7 +102,13 @@
             this.cameraDirection = new Vector3(0, 0, 0);
             this.cameraRotation = new Vector2(0, 0);
         }
-
+	
+	 // Collisions
+	private _collisionMask = -1;
+	public set setCollisionMask(mask: number) {
+	    this._collisionMask = mask && !isNaN(mask) ? mask : -1;

```

Comment:
not sure of this line
why not just this._collisionMask = !isNaN(mask) ? mask : -1;